lib/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@
 ##-*****************************************************************************
 
 # Include the source dirs to make the library root dir an include
-INCLUDE_DIRECTORIES(${ALEMBIC_LIBS_SOURCE_DIR})
+INCLUDE_DIRECTORIES(${ALEMBIC_LIBS_SOURCE_DIR} ${PROJECT_BINARY_DIR}/lib)
```

Comment:
Was this a problem with JUST PyAlembic or did some of the other libraries/utilies have problems finding the config file?